fr/index.html

```diff
@@ -0,0 +1,391 @@
+              >Netflix</a
+            >
+            et <a href="https://www.cnbc.com/2019/10/07/dominos-supreme-court.html" target="_blank">Domino’s Pizza</a>.
+            Vous verrez que le prix à payer pour ne pas être accessible peut être très élevé. Il existe
+            <a href="https://www.w3.org/WAI/policies/" target="_blank">de nombreuses lois</a> qui exigent des site web
+            qu’ils soient accessibles.
+          </p>
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">L'accessibilité ne sert qu'à éviter les poursuites judiciaires</h2>
+
+          <p>
+            L'accessibilité concerne d’abord vos utilisateurs. Bien sûr, il est important pour toute entreprise d'éviter
+            des poursuites judiciaires, mais la satisfaction des utilisateurs et la réputation de l'entreprise sont tout
+            aussi importantes.
+          </p>
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">
+            L'accessibilité ne consiste qu'à suivre les règles pour l’accessibilité des contenus web (WCAG)
+          </h2>
+
+          <p>
+            En général, cela commence par le
+            <a target="_blank" href="https://www.w3.org/Translations/WCAG20-fr/">WCAG</a>, mais dans la vie réelle, un
+            site web conforme peut ne pas être totalement accessible. L'accessibilité est une question de compréhension
+            des besoins de tous les utilisateurs, d'empathie et de sensibilisation.
+          </p>
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">L'accessibilité peut être obtenue en ajoutant uniquement des attributs ARIA</h2>
+
+          <p>
+            Bien sûr, <a href="https://www.w3.org/WAI/standards-guidelines/aria/" target="_blank">ARIA</a> constitue une
+            part importante de l'accessibilité, mais il ne faut jamais oublier la règle d'or d’ARIA:
+            <a href="https://w3c.github.io/using-aria/#rule1" target="_blank"
+              >N'utilisez pas ARIA que si vous en avez vraiment besoin</a
+            >. De nombreux attributs ARIA peuvent ne pas être nécessaires si
+            <a
+              href="https://developer.mozilla.org/fr/docs/Learn/Accessibility/HTML#une_bonne_s%C3%A9mantique"
+              target="_blank"
+              >des balises HTML sémantiques</a
+            >
+            sont utilisées à la place.
+          </p>
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">Nous n'avons pas besoin d'être conformes</h2>
+
+          <p>
+            Peut-être que vous en avez besoin. Il existe un certain nombre de lois et de règlements qui peuvent exiger
+            qu'un site Web respecte les principes d'accessibilité. Cela vaut assurément la peine de vérifier.
+          </p>
+
+          <a href="https://www.w3.org/WAI/policies/" target="_blank"
+            >Plus de détails <span class="visually-hidden"></span
```

Comment:
@MarionCouesnon could you please add a myth name to `visually-hidden`.

Reply:
Done with 77c2a05d2459067877887858494997c996167cdd